myhpi/static/js/myHPI.js

```diff
@@ -54,6 +54,29 @@ const toggleElementVisibilityOnScroll = (minScrollPosition = 0) => {
     previousScrollPosition = currentScrollPosition
 }
 
+const localizeLastPublished = () => {
```

Comment:
Django supports activating a current timezone for the user natively, see https://docs.djangoproject.com/en/4.2/topics/i18n/timezones/#selecting-the-current-time-zone

I would prefer using that over a JS fix

Reply:
thats not possible, as the browser does not communicate the local timezone to the server. However, django would then use the default timezone. For me that behaviour would be fine